lib/ansible/modules/database/postgresql/postgresql_db.py

```diff
@@ -88,7 +88,18 @@
       - The database state
     required: false
     default: present
-    choices: [ "present", "absent" ]
+    choices: [ "present", "absent", "dump", "restore" ]
+    backup:
+      - Back up to a destination, requires target as well. Supported formats are
+        .gz, .tar, .bz, .xz, and .sql
+  restore:
+    version_added: "2.2"
+    description:
+      - Restore from a destination. See backup for supported options
+  target:
+    version_added: "2.2"
+    description:
+      - File to back up or restore from.
```

Comment:
Agreed. I didn't even realize the discrepancy when taking over https://github.com/ansible/ansible-modules-core/pull/2752